src/test/java/io/cdap/plugin/cloud/vision/action/OfflineImageExtractorActionTest.java

```diff
@@ -46,93 +46,166 @@
-  private static final String SAFE_SEARCH_IMAGE_FILE_PATH = String.format(RESULT_PATH_PATTERN, PATH, SAFE_IMAGE_NAME);
-  private static final String TEXT_IMAGE_FILE_PATH = "gs://cloud-samples-data/vision/ocr/sign.jpg";
-  private static final String WEB_DETECTION_IMAGE_FILE_PATH = "gs://cloud-samples-data/vision/web/carnaval.jpeg";
+
+  private static final String LOCAL_IMAGE_FOLDER_PATH = "examples/images/jpg";
+
+  private static final String CROP_HINTS_IMAGE_NAME = "bubble.jpg";
+  private static final String FACE_IMAGE_NAME = "faces.jpg";
+  private static final String LABELS_IMAGE_NAME = "setagaya.jpg";
+  private static final String LANDMARKS_IMAGE_NAME = "st_basils.jpg";
+  private static final String LOGOS_IMAGE_NAME = "google_logo.jpg";
+  private static final String OBJECT_IMAGE_NAME = "fallon-michael-8LKQfBumjMo.jpg";
+  private static final String PROPERTIES_IMAGE_NAME = "bali.jpg";
+  private static final String SAFE_SEARCH_IMAGE_NAME = "keyur-nandaniya-oEgiJNbYw8w.jpg";
+  private static final String TEXT_IMAGE_NAME = "sign.jpg";
+  private static final String WEB_DETECTION_IMAGE_NAME = "carnaval.jpg";
+
+  private static final String CROP_HINTS_IMAGE_FILE_PATH = BUCKET_NAME + "/" + IMAGE_PATH_IN_BUCKET
+          + "/" + CROP_HINTS_IMAGE_NAME;
+  private static final String FACE_IMAGE_FILE_PATH = BUCKET_NAME + "/" + IMAGE_PATH_IN_BUCKET
+          + "/" + FACE_IMAGE_NAME;
+  private static final String LABELS_IMAGE_FILE_PATH = BUCKET_NAME + "/" + IMAGE_PATH_IN_BUCKET
+          + "/" + LABELS_IMAGE_NAME;
+  private static final String LANDMARKS_IMAGE_FILE_PATH = BUCKET_NAME + "/" + IMAGE_PATH_IN_BUCKET
+          + "/" + LANDMARKS_IMAGE_NAME;
+  private static final String LOGOS_IMAGE_FILE_PATH = BUCKET_NAME + "/" + IMAGE_PATH_IN_BUCKET
+          + "/" + LOGOS_IMAGE_NAME;
+  private static final String OBJECTS_IMAGE_FILE_PATH = BUCKET_NAME + "/" + IMAGE_PATH_IN_BUCKET
+          + "/" + OBJECT_IMAGE_NAME;
+  private static final String PROPERTIES_IMAGE_FILE_PATH = BUCKET_NAME + "/" + IMAGE_PATH_IN_BUCKET
+          + "/" + PROPERTIES_IMAGE_NAME;
+  private static final String SAFE_SEARCH_IMAGE_FILE_PATH = BUCKET_NAME + "/" + IMAGE_PATH_IN_BUCKET
+          + "/" + SAFE_SEARCH_IMAGE_NAME;
+  private static final String TEXT_IMAGE_FILE_PATH = BUCKET_NAME + "/" + IMAGE_PATH_IN_BUCKET + "/"
+          + TEXT_IMAGE_NAME;
+  private static final String WEB_DETECTION_IMAGE_FILE_PATH = BUCKET_NAME + "/" + IMAGE_PATH_IN_BUCKET
+          + "/" + WEB_DETECTION_IMAGE_NAME;
+
   private static final String RESULT_FILE_NAME = "output-1-to-1.json";
 
   private static final String BATCH_SIZE = "20";
 
   private static final OfflineImageExtractorActionConfig CONFIG = new OfflineImageExtractorActionConfig(
-    SERVICE_ACCOUNT_FILE_PATH,
-    ImageFeature.FACE.getDisplayName(),
-    null,
-    null,
-    true,
-    null,
-    null,
-    BATCH_SIZE
+          SERVICE_ACCOUNT_FILE_PATH,
```

Comment:
can you undo the indentation changes in this file?

Reply:
fixed